.github/CONTRIBUTING.md

```diff
@@ -2,93 +2,68 @@
 
 ## Getting Started
 
-> TODO
+## Layout
 
-1. Run the Brewfile bundle
+This is a single top-level namespace filled with packages.  Each directory is potentially a package. Binary builds are done on packages with a main subpackage.
 
-    ```sh
-    brew bundle
-    ```
+## Building Locally
 
-1. Install go (currently 1.9)
+### Go Version
 
-    You may want to use a go version manager. [goenv](https://github.com/syndbg/goenv), [asdf](https://github.com/kennyp/asdf-golang), [gimme](https://github.com/travis-ci/gimme) are all options.
+Requires Go `>= 1.12.6`.
 
-    ```sh
-    go version
-    # go version go1.9 darwin/amd64
-    ```
 
-1. Instal dep (currently the version manager)
+1. Install go (currently 1.12)
 
-    ```sh
-    curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
-    ```
+    You may want to use a version manager.  
 
-1. Install gomock
+    [asdf](https://github.com/kennyp/asdf-golang)
```

Comment:
I may write up my current multi-version setup, which has worked out great. asdf on its own doesn't do enough manipulation of the environment vars to have a safe multi-version environment, I had to use dotenv as well, and multiple GOROOT

Reply:
To avoid these issue you can remove GOROOT env variable
https://dave.cheney.net/2013/06/14/you-dont-need-to-set-goroot-really

Reply:
except then it just defaults to `~/go` - especially if you're switching between <1.11 and >= 1.11. in any case, my approach worked great.

Reply:
Are you going to update the PR branch with your approach?   Personally, I would only support 1.11+ that does not require GOROOT.

Reply:
might not be worth adding here since you'd only use one and its probably out of scope of this repo, I might write my own blog post.